web/core/layouts/auth-layout/project-wrapper.tsx

```diff
@@ -162,7 +162,7 @@ export const ProjectAuthWrapper: FC<IProjectAuthWrapper> = observer((props) => {
   // check if the project info is not found.
   if (!loader && !projectExists && projectId && !!hasPermissionToCurrentProject === false)
     return (
-      <div className="container grid h-screen place-items-center bg-custom-background-100">
+      <div className="grid h-screen place-items-center bg-custom-background-100">
```

Comment:
@NarayanBavisetti Why is this change required?

Reply:
I made that change, when the sidebar is collapsed, the alignment of the error state is incorrect due to this.